config.tfvars

```diff
@@ -58,6 +58,19 @@ max_cluster_capacity = 5
 #  }
 #]
 
+################################################################################
+# Osquery settings. Atlassian only!
+################################################################################
+
+# The secret needs to be available in the AWS region you deploy to. Terraform DOES NOT
+# create the secret. It should be just the secret name, not the full ARN.
+# Providing the secret name enables osquery installation in the nodegroup launch template.
+# osquery_fleet_enrollment_secret_name = "<FLEET-ENROLLMENT_SECRET-NAME>"
+
+# The value of OSQUERY_ENV that will be used to send logs to Splunk. It should not be something like “production”
+# or “prod-west2” but should instead relate to the product, platform, or team. Defaults to osquery_dc_e2e_tests
+# osquery_env = "example_osquery_env"
```

Comment:
We can either skip listing the default in the comment or include the value in the default commented value.

```suggestion
# osquery_env = "osquery_dc_e2e_tests"
```